lua/lspconfig/configs/nextflow_ls.lua

```diff
@@ -0,0 +1,37 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'java', '-jar', 'nextflow-language-server-all.jar' },
+    filetypes = { 'nextflow' },
+    root_dir = function(fname)
+      return util.root_pattern('nextflow.config')(fname) or util.find_git_ancestor(fname)
+    end,
+    settings = {
+      nextflow = {
+        files = {
+          exclude = { '.git', '.nf-test', 'work' },
+        },
+      },
+    },
+  },
+  docs = {
+    description = [[
+https://github.com/nextflow-io/language-server
+
+Requirements:
+ - Java 17+
+
+`nextflow_ls` can be installed by following the instructions [here](https://github.com/nextflow-io/language-server.git#development).
```

Comment:
```suggestion
`nextflow_ls` can be installed by following the instructions [here](https://github.com/nextflow-io/language-server#development).
```

Another copy pasta issue with the repo link

Reply:
Dang, the GitHub web interface messed up the commits. I was hoping to be able to finish up this PR on my phone. I'll do the rebasing and fixing of the commit history in a couple hours when I'm next at a computer. Sorry!